gradle-plugin/src/main/java/com/squareup/anvil/plugin/AnvilPlugin.kt

```diff
@@ -134,6 +134,14 @@ open class AnvilPlugin : KotlinCompilerPluginSupportPlugin {
         project.dependencies.add("implementation", "$GROUP:annotations:$VERSION")
       }
     }
+
+    // Create a configuration for collecting CodeGenerator dependencies
+    val anvilConfiguration = project.configurations.maybeCreate("anvil").apply {
+      description = "This configuration is used for dependencies with Anvil CodeGenerator implementations."
+    }
+    // Make the kotlin compiler classpath extend it to pick up our extra generators
+    project.configurations.getByName("kotlinCompilerPluginClasspath")
```

Comment:
Use the constant `PLUGIN_CLASSPATH_CONFIGURATION_NAME`, it's already used in this file.

Reply:
Ah nice, fixed